### PR TITLE
#30: Quick conversion screen

### PR DESCRIPTION
**Pull request by mdrlzy**
- #30 
- #35
- #38
- #40

### CODE REVIEW

File: app/src/main/java/dev/arkbuilders/rate/presentation/quick/QuickScreen.kt [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+            keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
+            maxLines = 1,
+        )
+        OutlinedButton(
+            modifier = Modifier.padding(8.dp),
+            onClick = {
+                viewModel.onExchange()
+            }
+        ) {
+            Text(text = "Exchange", fontSize = 20.sp)
```
Comment (kirillt) [inline app/src/main/java/dev/arkbuilders/rate/presentation/quick/QuickScreen.kt:166]: Let's call the button "Convert" or "Calculate"

**Comment by kirillt** [comment]
@mdrlzy Could you also implement these features in this PR?
- #29
- #35
- #38
- #40

**Comment by mdrlzy** [comment]
- #29 

Impossible to implement, we need a prepared file or API with a description (flag, pictogram) for each currency

**Comment by kirillt** [comment]
Looks good. Couple of minor enhancements would be nice:
- [ ] Default screen should be "quick mode"
- [ ] Quick mode should have switch between views (let's call tag cloud "3D" in it)
       We can remove the preference from Preferences screen and just remember state of the switch.
- [ ] Additional "detailed" view would be great, i.e. list of the following format:
       `[CODE] [Full currency name] [Country]` for fiat
       `[CODE] [Blockchain name]` for crypto
       List could be formatted as 1 or 2 columns.

**Comment by kirillt** [comment]
Also really small bug (appeared long time ago):
- [ ] In currency search dialog, user can input new line

**Comment by kirillt** [comment]
Only 1 minor UI issue left:
- [ ] "Convert XXX to:" label is too close to the input, I think it should be aligned to be right near the list of base currencies

**Review by kirillt — APPROVED** [review]
Looks really nice!

**Pull request merged**